### ISSUE TITLE
Applying for 21C Badges on Staging

### ISSUE THREAD

**Issue by IBGMozilla** [mozilla/learning.mozilla.org#2406]
When walking through the demo on https://learning.mofostaging.net/en-US/badges for the 21Century badges (problem-solving, creativity, communication, collaboration), I could not "apply" (apply button was in grey) even after adding the required evidence. This should be fixed so the badges are live and we can apply for them as we can for the other web literacy badges. 

@Pomax @anmechung

**Comment by Pomax** [comment]
I'm going to look into this, but: why are there way more than three tasks for these badges? I remember there was the explicit mention that badges would only have three tasks so that we could hardcode the task label to say "Complete the following three tasks and submit the required evidence:".

**Comment by IBGMozilla** [comment]
This was how these were developed so I added all of the questions/tasks in
as they were initially conceived. I spoke to An-Me about this yesterday and
we think that we should say, "complete three of the following tasks". Is
that doable?

**Comment by Pomax** [comment]
Not really, we don't have a label "per badge", this is a universal label. As such, if we say "complete three of the following tasks" for badges that have exactly three tasks, that's going to be a curious label. 

That said, the format for the four original badges will still needs to be updated to match the formatting used by the new badges (I can help with that if necessary but it's mostly a matter of "delete what's there now, and then put it back in in the same way as has been done for the new badges" so I don't think you'll need my help for that?)

**Comment by Pomax** [comment]
basically we've set it up right now so that the label says "three", and the button does not unlock until you've done all of them, because all badges had three requirements. If we're going to change that, then we need to go back into the code and change how the form itself works, which would be a fair bit of work.
